src/http-client.ts

```diff
@@ -75,6 +76,10 @@ export class HttpClient {
         let status = response.status;
         let correlationId = response.headers?.[correlationIdHeaderName];
         let body = response.data;
+        let error: IAxiosResponseError = {
+          ...res,
+          text: response.request.response
```

Comment:
Please correct me if I'm wrong, but I don't think this will contain PII. This is the missing field that Dave's tests look for.